odysseus/convertFtbQuests.ts

```diff
@@ -379,20 +393,49 @@ export const convertFtbQuests = async (input: QuestInputFileSystem, output: Ques
                 const inferData = () => {
                     if (taskIds.length === 1) {
                         const task = tasks[taskIds[0]];
-
-                        if (task.type === 'heracles:item') {
-                            return {
-                                title: `Gather ${task.item}`,
-                                icon: task.item.startsWith('#') ? undefined : task.item
-                            };
+                        let title = undefined;
+                        let icon = undefined;
+                        if (task.title) title = task.title;
+                        if (task.icon) icon = task.icon;
+                        switch (task.type) {
```

Comment:
I feel like this should be a map rather than a switch statement.

Reply:
uhh, i can see how that might work, but record of what type to what type?

Reply:
Key would be `HeraclesQuestTask['type']`, the value would probably some type that has functions to convert the the title etc.

Reply:
function'd in 4e093f9